frontend/projects/admin/src/app/event/email-log/email-log.component.html

```diff
@@ -1 +1,69 @@
-<p>email-log works!</p>
+<div class="container" *ngIf="mailLog$ | async as mailLog">
+  <h1 class="my-3" translate="admin.event.mail-log.title"></h1>
+  <p translate="admin.event.mail-log.legend"></p>
+  <hr />
+  <div class="fs-2 my-3">
+    <svg-icon key="mail" size="xxl"></svg-icon>
+    <span translate="admin.event.mail-log.emails"></span>
+  </div>
+
+  <div class="input-group">
+    <button class="btn btn-outline-secondary" type="button" (click)="search()">
```

Comment:
this button should be on the right side of the group and should also have a "search" text